### PR TITLE
Docker host updates

### PR DESCRIPTION
**Pull request by rvagg**
This is live now. Also included instructions on how to do the manual prep bits for making image changes.

### CODE REVIEW

File: doc/non-ansible-configuration-notes.md [hunk 1]
```diff
@@ -67,6 +67,45 @@ The jenkins-workspace hosts are setup as standard Node.js nodes but are only giv
...
 * Ensure that the [node-coverity-daily](https://ci.nodejs.org/job/node-daily-coverity/configure) job matches the path used in its explicit `PATH` setting
 
+## Docker hosts
+
+The hosts that run Docker images for "sharedlibs", Alpine Linux and a few other dedicated systems (hosts identified by `grep _docker-x64- inventory.yml`) don't have Docker image reload logic built in to Ansible. Changes to Docker images (adding, deleting, modifying) involve some manual preparation.
+
+The general steps are:
+
+1. Stop the concerned Jenkins systemd service(s)
+2. Disable the concerned Jenkins systemd service(s)
```
Comment (mhdawson) [inline doc/non-ansible-configuration-notes.md]: Could we add the actual commands lines for these in 1 and 2?

Comment (mhdawson) [reply doc/non-ansible-configuration-notes.md]: actually for 1-3 since we have it for 4 and 5 it would make it a no-bainer to follow the instructions.

Comment (mhdawson) [reply doc/non-ansible-configuration-notes.md]: I see it is below, maybe just integrate with this bullet point list, but no strong feeling on that front.

**Comment by rvagg** [comment]
added more inline docs for the docker cleanup instructions

merged

**Pull request closed without merging**